src/typeDefs/User.js

```diff
@@ -54,6 +54,7 @@ export default gql`
     vehicle: Vehicle!
     employeeType: String!
     favouriteBranch: [FavouriteBranch!]!
+    employementBranch: String
```

Comment:
employment* typo

Reply:
fixed in 308cd39bb6ce11bb36156300bdafb85e14f06a55
**with some update in typeDefs for user and dispatchService**